Microservices/event-service/src/main/java/app/sportahub/eventservice/exception/event/EventAlreadyExistsException.java

```diff
@@ -0,0 +1,13 @@
+package app.sportahub.eventservice.exception.event;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+@ResponseStatus(code = HttpStatus.CONFLICT, reason = "Event with this id already exists.")
```

Comment:
If the even name is unique, you should be saying that `Event with this name already exists.` instead of Event with this **id** already exists.

Reply:
Done